forge-gui/res/cardsfolder/r/repopulate.txt

```diff
@@ -1,7 +1,7 @@
 Name:Repopulate
 ManaCost:1 G
 Types:Instant
-A:SP$ ChangeZoneAll | Origin$ Graveyard | Destination$ Library | ValidTgts$ Player | TgtPrompt$ Select target player | ChangeType$ Creature | Shuffle$ True | SpellDescription$ Shuffle all creature cards from target player's graveyard into that player's library.
+A:SP$ ChangeZoneAll | Origin$ Graveyard | Destination$ Library | ValidTgts$ Player | TgtPrompt$ Select target player | ChangeType$ Creature | Shuffle$ True | AILogic$ ManifestCreatsFromGraveyard | SpellDescription$ Shuffle all creature cards from target player's graveyard into that player's library.
```

Comment:
So why did you end up using this AILogic? `ManifestCreatsFromGraveyard` I guess this at least slightly improves the play of Repopulate I wonder if it really should be using its own AILogic, something like "Replenish my library with something good. Or prevent my opponent for utilizing their graveyard as a resource"

Reply:
oh yea using shuffle to screw up opponents primed top deck would be advanced play :)
I'm not sure we've done that before in any other effects that could but feel free to try
the main problem is the AI doesn't have that great of memory helpers to know an opponent did something like that though